mantle/cmd/kola/devshell.go

```diff
@@ -113,32 +113,13 @@ func runDevShellSSH(builder *platform.QemuBuilder, conf *v3types.Config) error {
 	}
 	sshPubKey := v3types.SSHAuthorizedKey(strings.TrimSpace(string(sshPubKeyBuf)))
 
-	readinessSignalChan := "coreos.devshellready"
-	signalReadyUnit := fmt.Sprintf(`[Unit]
-Requires=dev-virtio\\x2dports-%s.device
-OnFailure=emergency.target
-OnFailureJobMode=isolate
-After=systemd-user-sessions.service
-After=sshd.service
-[Service]
-Type=oneshot
-RemainAfterExit=yes
-ExecStart=/bin/echo ready
-StandardOutput=file:/dev/virtio-ports/%[1]s
-[Install]
-RequiredBy=multi-user.target`, readinessSignalChan)
-	signalPoweroffUnit := fmt.Sprintf(`[Unit]
-Requires=dev-virtio\\x2dports-%s.device
-OnFailure=emergency.target
-OnFailureJobMode=isolate
-Conflicts=reboot.target
-[Service]
-Type=oneshot
-RemainAfterExit=yes
-ExecStop=/bin/echo poweroff
-StandardOutput=file:/dev/virtio-ports/%[1]s
-[Install]
-WantedBy=multi-user.target`, readinessSignalChan)
+	// Await sshd startup
+	journalConf, journalPipe, err := builder.VirtioJournal("-u sshd MESSAGE_ID=7d4958e842da4a758f6c1cdc7b36dcc5")
```

Comment:
Ahh OK, I see this is `SD_MESSAGE_UNIT_STARTING`. Hmm... don't we want to use `SD_MESSAGE_UNIT_STARTED` for this instead?

Also, let's add a comment mentioning the symbolic name (or maybe just link to the systemd source)?

Reply:
Good suggestion, done!